dags/atd_service_bot_issues_to_socrata.py

```diff
@@ -62,16 +63,26 @@
     dagrun_timeout=timedelta(minutes=60),
     tags=["repo:atd-service-bot", "socrata", "github"],
     catchup=False,
+    render_template_as_native_obj=True,
 ) as dag:
+    @task(
+        task_id="get_env_vars",
+        execution_timeout=timedelta(seconds=30),
+    )
+    def get_env_vars():
+        from utils.onepassword import load_dict
+        env_vars = load_dict(REQUIRED_SECRETS)
+        return env_vars
```

Comment:
by default, anything returned from a python `task` will be set into the XComs key called `return_value`.